### PR TITLE
gh-action: fix workflow error

### PR DESCRIPTION
**Pull request by bpradipt**
Fixes: #633

Tested locally using nektos/act

### CODE REVIEW

**Comment by bpradipt** [comment]
Found a cool s/w to test gh-actions locally - https://github.com/nektos/act
```
$ act -s QUAY_USERNAME=<username> -s QUAY_PASSWORD=<password> -W .github/workflows/image.yaml

[snip]
[image/build and push-3]   ✅  Success - Main Build and push image
[image/build and push-3] ⭐ Run Post Login to quay Container Registry
[image/build and push-3]   🐳  docker exec cmd=[node /var/run/act/actions/docker-login-action@v2/dist/index.js] user= workdir=
| [command]/usr/bin/docker logout quay.io
| Removing login credentials for quay.io
[image/build and push-3]   ✅  Success - Post Login to quay Container Registry
[image/build and push-3] ⭐ Run Post Set up Docker Buildx
[image/build and push-3]   🐳  docker exec cmd=[node /var/run/act/actions/docker-setup-buildx-action@v2/dist/index.js] user= workdir=
[image/build and push-3]   ❓  ::group::Removing builder
| [command]/usr/bin/docker buildx rm builder-9a609f99-dd47-4d1e-a85f-7f579c3aa94c
| builder-9a609f99-dd47-4d1e-a85f-7f579c3aa94c removed
[image/build and push-3]   ❓  ::endgroup::
| Cleaning up credentials
[image/build and push-3]   ✅  Success - Post Set up Docker Buildx
[image/build and push-3] ⭐ Run Post Setup Golang version 1.18
[image/build and push-3]   🐳  docker exec cmd=[node /var/run/act/actions/actions-setup-go@v3/dist/cache-save/index.js] user= workdir=
[image/build and push-3]   ✅  Success - Post Setup Golang version 1.18
[image/build and push-3] 🏁  Job succeeded
```

**Review by yoheiueda — APPROVED** [review]
LGTM

File: .github/workflows/image.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
         arches:
```
Comment (jtumber-ibm) [inline .github/workflows/image.yaml:24]: Is it clearer to do

```yaml
      matrix:
        include:
          - type: dev
            arches: linux/amd64
        include:
          - type: release
            arches: linux/amd64,linux/s390x

Comment (bpradipt) [reply .github/workflows/image.yaml]: Done. PTAL

**Review by jtumber-ibm — APPROVED** [review]
lgtm

**Comment by jtumber-ibm** [comment]
@bpradipt 
`unexpected status: 401 UNAUTHORIZED` in the image job? Credentials need updating?

**Pull request merged**